domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientIT.java

```diff
@@ -1751,15 +2227,16 @@ public void endTx(AsyncResult<SQLConnection> conn, Handler<AsyncResult<Void>> do
   }
 
   @Test
-  public void execute(TestContext context) {
+  public void executeOK(TestContext context) {
     Async async = context.async();
     JsonArray ids = new JsonArray().add(randomUuid()).add(randomUuid());
     insertXAndSingleQuotePojo(context, ids)
     .execute("DELETE FROM tenant_raml_module_builder.foo WHERE id='" + ids.getString(1) + "'", res -> {
       assertSuccess(context, res);
-      context.assertEquals(1, res.result().getUpdated());
+      context.assertEquals(1, res.result().rowCount());
       async.complete();
     });
+    async.await(1000);
```

Comment:
```suggestion
```
This is not needed and can be too short when Jenkins has high load. We have a default at the beginning of PostgresClientIT that is sufficient: `Timeout rule = Timeout.seconds(15);`

Reply:
Yes sir!